app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -213,10 +222,29 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
   }
 
   fun showSimpleVideo() {
+    //Hard-coding to show this simple video view exists because, as of now,
+    // we are not sure if we will have variations of this training video.
+    // We should improve this implementation later.
```

Comment:
Can you also mention how we can improve to convey the kind of improvement that we should make? So the maintainer could have some context about the problem that we are attempting to solve?